src/components/Chat/InputMessageV2.tsx

```diff
@@ -158,7 +363,7 @@ const InputMessageV2 = ({
 
     Alert.alert(
       '',
-      type === 'ANONYMOUS'
+      type === ANONYMOUS
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The comparison `type === ANONYMOUS` has been introduced instead of `type === 'ANONYMOUS'`. This is a good change as it reduces the chance of typos and makes the code easier to maintain.



<!-- This is an auto-generated comment by OSS CodeRabbit -->